caliban/hooks/git_commit_hooks.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Git commit prerun and prebuild hooks.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from typing import Any, Dict, List, NamedTuple, NewType, Optional, Union
+
+import sys
+from absl import logging
+import docker
+from git import Repo
+
+
+def get_hash_from_image(container_id):
+  """Load git hash from docker image label."""
+  client = docker.from_env()
```

Comment:
can you do this with a shell call? I'd rather not add another dependency; the docker python dep was pretty unhelpful for us before, so we decided not to use it.

Reply:
Yep, I've removed the `docker` dependency and done this with a subprocess call in the latest version.